dev/storage_traits.h

```diff
@@ -1,24 +1,18 @@
-#include <type_traits>  //  std::is_same, std::enable_if, std::true_type, std::false_type, std::integral_constant
+#pragma once
+
 #include <tuple>  //  std::tuple
 
-#include "type_traits.h"
-#include "storage_lookup.h"
+#include "functional/cxx_universal.h"
+#include "functional/cxx_polyfill.h"
 #include "tuple_helper/tuple_filter.h"
 #include "tuple_helper/tuple_transformer.h"
+#include "type_traits.h"
+#include "storage_lookup.h"
 
 namespace sqlite_orm {
 
     namespace internal {
 
-        template<class... Ts>
-        struct storage_impl;
-
-        template<class T, bool WithoutRowId, class... Cs>
-        struct table_t;
-
-        template<class A, class B>
-        struct foreign_key_t;
-
         namespace storage_traits {
```

Comment:
@fnc12 Do you want to keep the `storage_traits` nested namespace?

Reply:
yes